app/file-detail/template.hbs

```diff
@@ -187,8 +187,18 @@
                             <th class='col-md-4'>Version ID</th>
                             <th class='col-md-6'>Date</th>
                             <th colspan='2' class='col-xs-2'>Download</th>
-                            <th class='hidden-md hidden-sm hidden-xs'>MD5</th>
-                            <th class='hidden-md hidden-sm hidden-xs'>SHA2</th>
+                            <th class='hidden-md hidden-sm hidden-xs'>
+                                MD5
+                                <i class='fa fa-question-circle'>
```

Comment:
Let's try using [{{fa-icon}}](https://github.com/martndemus/ember-font-awesome#using-the-add-on) here. Not sure if you can use it as a block helper to wrap {{bs-popover}}, though.